bppy/model/event_selection/simple_event_selection_strategy.py

```diff
@@ -37,10 +37,13 @@ def selectable_events(self, statements):
                     possible_events = {x for x in possible_events if x not in statement.get('block')}
         return possible_events
 
-    def select(self, statements):
+    def select(self, statements, external_events_queue=[]):
         selectable_events = self.selectable_events(statements)
         if selectable_events:
             return random.choice(tuple(selectable_events))
```

Comment:
Something I noticed.
If there are no possible selected events, random.choice will raise an exepction.
Example:
`import random
selectable_events = set()
print(type(selectable_events))
event = random.choice(tuple(selectable_events))`

Reply:
```suggestion
        if len(possible_events) > 0:
            return possible_events
        else:
            return None
```

Reply:
@adielashrov yeah. that's why we check if it is empty in the select method.
keep in mind that:
`
if []:
    print("has events")
else:
    print("no events")
`
return "no events"